taskvine/src/worker/vine_worker.c

```diff
@@ -1092,6 +1101,19 @@ static int handle_manager(struct link *manager)
 			reset_idle_timer();
 			hash_table_insert(current_transfers, strdup(filename), strdup(transfer_id));
 			debug(D_VINE, "Insert ID-File pair into transfer table : %s :: %s", filename, transfer_id);
+		} else if (sscanf(line,
+					   "puturl_now %s %s %" SCNd64 " %o %s",
+					   source_encoded,
+					   filename_encoded,
+					   &length,
+					   &mode,
+					   transfer_id) == 5) {
+			url_decode(filename_encoded, filename, sizeof(filename));
+			url_decode(source_encoded, source, sizeof(source));
+			r = do_put_url_now(filename, length, mode, source);
+			reset_idle_timer();
+			hash_table_insert(current_transfers, strdup(filename), strdup(transfer_id));
+			debug(D_VINE, "Insert ID-File pair into transfer table : %s :: %s", filename, transfer_id);
```

Comment:
Is this a debug statement for you?

Reply:
I suppose so, it is just mirroring the existing puturl handler. I will remove them